scripts/update/Updater.php

```diff
@@ -315,6 +315,6 @@ public function update($currentVersion)
             call_user_func(new RegisterPciAudioRecording(), ['0.4.0']);
             $this->setVersion('4.6.0');
         }
-        $this->skip('4.6.0', '4.6.5');
+        $this->skip('4.6.0', '4.7.0');
```

Comment:
You need to re-register the PCI at each PCI version update, otherwise it will not get updated in the data folder:

```
call_user_func(new RegisterPciAudioRecording(), ['0.5.0']);
```

Reply:
Fixed